R/stattle.R

```diff
@@ -14,9 +13,8 @@
 #' 
 #' @examples 
 #' \dontrun{
-#' TOKEN <- "insert-your-token-here"
-#' results <- stattle(TOKEN, 
-#'                    sport="hockey", 
+#' setToken("insert-your-token-here")
+#' results <- stattle(sport="hockey", 
```

Comment:
I'm not a huge fan of the "stattle" method name.

It's not a term anyone (even us) really know.

So, the package is "StattleshipR" ... should the method describe what it does? 

Fetch? Process? Perform? MakeDataFrame?

I err on "parse" or "perform" when in doubt.

What's a common "do_it" in R?